include/core/parameters_lagrangian.h

```diff
@@ -604,6 +604,37 @@ namespace Parameters
       parse_parameters(ParameterHandler &prm);
     };
 
+    /**
+     * @brief ParticleRayTracing - Defines parameters used to construct particle
+     * ray tracing elements.
+     */
+    template <int dim>
+    struct ParticleRayTracing
+    {
+      Point<dim>     starting_point;
+      Tensor<1, dim> first_direction;  // This is a unit vector
+      Tensor<1, dim> second_direction; // This is a unit vector
+      Tensor<1, dim> photon_displacement_vector;
+
+      // Number of photon inserted in each direction
```

Comment:
Is it not a better idea to make these lists instead?
I understand you only have two directions, but nothing could prevent a 3D insertion one day. We have learnt from this mistake in the past. I would have two std::vectors number_of_photon_per_direction and step_between_photos_per_direction